bpf/Makefile

```diff
@@ -213,6 +213,10 @@ tests/bpf_ct_tests.o: tests/bpf_ct_tests.ll
 	@$(ECHO_CC)
 	$(QUIET) ${LLC} ${LLC_FLAGS} -filetype=obj -o $@ $(patsubst %.o,%.ll,$@)
 
+.PHONY: go_prog_test
+go_prog_test: $(BPF_TEST)
+	sudo go test ./tests/prog_test
```

Comment:
From the development VM:
```
vagrant@k8s1:~/go/src/github.com/cilium/cilium/bpf$ sudo go test ./tests/prog_test
package github.com/cilium/cilium/bpf/tests/prog_test: build constraints exclude all Go files in /home/vagrant/go/src/github.com/cilium/cilium/bpf/tests/prog_test
```
Same error if I run `make -C bpf go_prog_test` from the root of the Cilium sources.

Reply:
Thanks for testing this! I'll boot the dev VM and figure out how to solve it. (Only tested it locally.)